hypervisor/src/vm.rs

```diff
@@ -190,7 +190,7 @@ pub enum HypervisorVmError {
     /// Assert virtual interrupt error
     ///
     #[error("Failed to assert virtual Interrupt: {0}")]
-    AsserttVirtualInterrupt(#[source] anyhow::Error),
+    AssertVirtualInterrupt(#[source] anyhow::Error),
```

Comment:
This is probably an api break, but I think it'd be correct.

Reply:
The change is fine. Don't worry about the breakage since all users are internal.